lear-db/lear_db.sql

```diff
@@ -339,7 +339,10 @@ CREATE TABLE public.filings (
     submitter_id integer,
     colin_event_id integer,
     status character varying(10),
-    payment_completion_date timestamp with time zone
```

Comment:
Why is this part of the PR?

Reply:
Yes, this file has been updated by another PR and is no longer needed. Removed from change set in new PR (rebasing is fubared)